__tests__/bookmarks.ts

```diff
@@ -568,56 +573,235 @@ describe('mutation moveBookmark', () => {
-    unreadOnly ? ', unreadOnly: true' : ''
-  }${listId ? `, listId: "${listId}"` : ''}) {
-    pageInfo {
-      endCursor
-      hasNextPage
-    }
-    edges {
-      node {
-        id
-        source {
-          id
-          name
-          image
-          public
+  const QUERY = `
+    query BookmarksFeed($first: Int, $now: DateTime, $listId: ID, $reminderOnly: Boolean, $unreadOnly: Boolean) {
+      bookmarksFeed(first: $first, now: $now, listId: $listId, reminderOnly: $reminderOnly, unreadOnly: $unreadOnly) {
+        pageInfo {
+          endCursor
+          hasNextPage
+        }
+        edges {
+          node {
+            id
+            bookmark {
+              remindAt
+              list {
+                id
+                name
+              }
+            }
+            source {
+              id
+              name
+              image
+              public
+            }
+            tags
+          }
         }
-        tags
       }
     }
-  }
-}`;
+  `;
 
   it('should not authorize when not logged in', () =>
-    testQueryErrorCode(
-      client,
-      {
-        query: QUERY(),
-      },
-      'UNAUTHENTICATED',
-    ));
+    testQueryErrorCode(client, { query: QUERY }, 'UNAUTHENTICATED'));
+
+  const saveBookmarkFixtures = (userId = loggedUser) =>
+    saveFixtures(
+      con,
+      Bookmark,
+      bookmarksFixture.map((b) => ({ ...b, userId })),
+    );
 
   it('should return bookmarks ordered by time', async () => {
     loggedUser = '1';
     await saveFixtures(con, Bookmark, bookmarksFixture);
-    const res = await client.query(QUERY(false, null, now, 2));
+    const res = await client.query(QUERY, { variables: { first: 2, now } });
     delete res.data.bookmarksFeed.pageInfo.endCursor;
     expect(res.data).toMatchSnapshot();
   });
 
+  it('should return bookmarks filtered with reminder only', async () => {
+    loggedUser = '1';
+    await saveFixtures(con, Bookmark, bookmarksFixture);
+    await con
+      .getRepository(Bookmark)
+      .update({ userId: '1', postId: 'p1' }, { remindAt: new Date() });
+    const res = await client.query(QUERY, {
+      variables: { first: 2, now, reminderOnly: true },
+    });
+    expect(res.data.bookmarksFeed.edges.length).toBeGreaterThan(0);
+    const isReminderOnly = res.data.bookmarksFeed.edges.every(
+      ({ node }) => node.bookmark.remindAt,
+    );
+    expect(isReminderOnly).toBeTruthy();
+  });
+
+  describe('plus user', () => {
+    beforeEach(async () => {
+      loggedUser = '5';
+      isPlus = true;
+      await con.getRepository(Settings).save({ userId: loggedUser });
+    });
+
+    it('should return bookmarks without list id only as plus user', async () => {
+      await saveBookmarkFixtures();
+      const list = await con
+        .getRepository(BookmarkList)
+        .save({ userId: loggedUser, name: 'Test' });
+      await con
+        .getRepository(Bookmark)
+        .update({ userId: loggedUser, postId: 'p3' }, { listId: list.id });
```

Comment:
Many updates in this file, might cause you headaches.
It might be better to store individual unique ones for this test, but non blocking.

Also noticed you never wait for nocks, might also cause some flakiness down the line (just noting for if it happens)